cmake/dependencies/PICSAR.cmake

```diff
@@ -15,7 +15,7 @@ function(find_picsar)
 
         if(NOT fetchedpicsar_POPULATED)
             FetchContent_Populate(fetchedpicsar)
-            add_subdirectory(${fetchedpicsar_SOURCE_DIR}/src/multi_physics ${fetchedpicsar_BINARY_DIR})
+            add_subdirectory(${fetchedpicsar_SOURCE_DIR}/src/multi_physics/QED ${fetchedpicsar_BINARY_DIR})
```

Comment:
Works for me, generally we could also add an option [here](https://github.com/ECP-WarpX/picsar/blob/9924a34515cb46a3be5ccb6cfcd939abde272c6e/src/multi_physics/CMakeLists.txt#L13) that sets if tests are included in the build or not.